src/tiled/tileseteditor.cpp

```diff
@@ -196,7 +196,13 @@ void TilesetEditor::addDocument(Document *document)
 
     TilesetView *view = new TilesetView(mWidgetStack);
     view->setTilesetDocument(tilesetDocument);
-    view->setZoomable(new Zoomable(this));
+
+    auto zoomable = new Zoomable(this);
+    auto fileName = tilesetDocument->fileName();
+    qreal zoom = Preferences::instance()->TilesetScale(fileName);
```

Comment:
Do you want that zoom in TilesetsDock window to be the same as in Edit Tileset window? I don't understand what "TilesetsDock displays the right zoom level" mean